modules/developers-guide/pages/concepts/data-centers.adoc

```diff
@@ -1,23 +1,23 @@
 = Decentralized data centers
 :keywords: Internet Computer,blockchain,protocol,replica,subnet,data center,smart contract,canister,developer
 :proglang: Motoko
-:IC: Internet Computer
+:IC: Internet Computer blockchain
 :company-id: DFINITY
 
-The {IC} is not physical hardware that exists in any physical location. 
-Instead, the {IC} combines computing resources provided by independently-operated data centers around the world. 
+The {IC} is not physical hardware that exists in any physical location.
+Instead, the {IC} combines computing resources provided by independently-operated data centers around the world.
 
-Unlike a public or private cloud, the {IC} is not owned and operated by a single private company. 
-Instead, the {IC} is a public utility with updates and operations that are managed through an algorithmic, decentralized governance system defined in the protocol.
+Unlike a public or private cloud, the {IC} is not owned and operated by a single private company.
+Instead, the {IC} is a public utility with updates and operations that are managed through an algorithmic, decentralized proof-of-stake (POS) governance system defined in the protocol.
 Its architecture enables multiple computers to operate like one, very powerful, virtual machine.
 
-The computers that make up the virtual machine are organized into sub-networks of data centers around the globe. 
+The computers that make up the virtual machine are organized into sub-blockchain networks of data centers around the globe.
```

Comment:
The nodes located in data centers around the globe that make up the Internet Computer are organised in subnet blockchains that in turn connect to each other using Chain Key cryptography.

Reply:
done